### DISCUSSION TITLE
Group in buckets

### DISCUSSION THREAD

**Discussion by smorkolama** [TelemetryDeck/Viewer#384] [Q&A]
Hello, thanks for building TelemetryDeck! I really love the dashboard and how easy it is to make graphs. However, I looked through the examples and TQL reference documentation but I could not find a solution for the following query, perhaps you have an idea?

I have a signal `persistentStoreLoad` that has an associated payload key `loadDuration` which contains a float value for how long it took to load the persistent store. I can list individual values just fine, but is there a way to group these values into buckets? This functionality probably hiding somewhere in there but I could not find it. Any hints on how to proceed?

**Comment by winsmith** [comment]
Hi, this is currently not possible, but I agree it probably should be. I'll create an issue and investigate when I have a bit of time.
